lib/todo-list.html

```diff
@@ -3,7 +3,7 @@
             <h1>To-Do</h1>
           </div>
           <div class="panel-body">
-            <h4 data-bind="visible: !tasks().length">You have nothing to do, great!</h4>
+            <h5 data-bind="visible: !tasks().length">You have nothing to do, great!</h5>
```

Comment:
This has nothing to do with this PR. But it's good you've brought it to attention. I think it should be a simple paragraph, not a header.